src/test/game/CommandLineUITest.java

```diff
@@ -17,17 +18,35 @@
     private ByteArrayOutputStream output;
     private  InputStream input;
     private CommandLineUI ui;
+    private English english;
 
     @Before
     public void setUp() {
         output = new ByteArrayOutputStream();
         input = new ByteArrayInputStream("".getBytes());
         ui = new CommandLineUI(new PrintStream(output), input);
+        english = new English();
+    }
+
+    @Test
+    public void askUserForLanguageSelection() {
+        ui.askUserForLanguageSelection();
+
+        assertTrue(output.toString().contains("Select your language:\nEnter '1' for English\nΕισαγάγετε '2' για τα Ελληνικά"));
```

Comment:
If the implementation of English changes, these tests will break. Perhaps you could implement your own TestLanguage to remove that dependency and to isolate these tests from English.